### PR TITLE
add wrapper for wallet import

### PR DESCRIPTION
**Pull request by yangjian102621**
add wrapper and unit test for wallet import

resolves https://github.com/filecoin-project/js-filecoin-api-client/issues/45

### CODE REVIEW

**Comment by alanshaw** [comment]
❤️ thanks @yangjian102621 - any chance you can update the API docs also? https://github.com/filecoin-project/js-filecoin-api-client/blob/master/API.md

**Comment by yangjian102621** [comment]
@alanshaw 
Yep, api docs updated and commited.

**Comment by yangjian102621** [comment]
@alanshaw 
Hi, is there any problem with my submit?

**Comment by alanshaw** [comment]
@yangjian102621 no I just haven't had a moment to re-review this - I'll get to it asap!

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ node_modules
 .nyc_output
 coverage
 dist
+/.eslintrc.js
```
Comment (alanshaw) [inline .gitignore]: This file is not in the repo - if you've added this for your own benefit please remove!

File: API.md [hunk 2]
```diff
@@ -1348,3 +1348,51 @@ console.log(info)
...
+|------|------|-------------|
+| addr | `KeyInfo`\|`KeyInfo[]` | KeyInfo(s) to import |
+| options | `Object` | Optional options |
+| options.signal | [`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) | A signal that can be used to abort the request |
+
+#### Returns
+
+| Type | Description |
+|------|-------------|
+| `Promise<Object>` | imported address(es) |
```
Comment (alanshaw) [inline API.md]:
```suggestion
| `Promise<String[]>` | imported address(es) |
```

File: .gitignore (hunk 1 above)
Comment (yangjian102621) [reply .gitignore]: Okay, i got it. 
thanks.

**Pull request merged**